### PR TITLE
Barnard59 refactor

### PR DESCRIPTION
**Pull request by ktk**
Implements issue #15 

@bergos I tested everything, please check if I forgot to delete steps that are not required here (as I took the cube one as a base)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,27 +1,34 @@
...
   "scripts": {
-    "postinstall": "mkdirp output",
-    "start": "dotenv ./start.sh --"
+    "start": "mkdir -p output && barnard59 run -v --pipeline=http://example.org/pipeline/toFile pipelines/main.ttl",
+    "to-file": "mkdir -p output && barnard59 run -v --pipeline=http://example.org/pipeline/toFile pipelines/main.ttl",
+    "file-to-store": "barnard59 run -v --pipeline=http://example.org/pipeline/fileToStore pipelines/main.ttl --variable=ENDPOINT_URL --variable=ENDPOINT_USER --variable=ENDPOINT_PASSWORD",
+    "file-to-store-dev": "barnard59 run -v --pipeline=http://example.org/pipeline/fileToStore pipelines/main.ttl",
+    "to-store-dev": "barnard59 run -v --pipeline=http://example.org/pipeline/toStore pipelines/main.ttl",
+    "fetch": "barnard59 run -v --pipeline=http://example.org/pipeline/fetch pipelines/fetch.ttl --variable=SSH_HOST --variable=SSH_PORT --variable=SSH_USER --variable=SSH_PRIVATE_KEY",
+    "fetch-dev": "barnard59 run -v --pipeline=http://example.org/pipeline/fetch pipelines/fetch.ttl",
```
Comment (bergos) [inline package.json]: I'm not sure if the fetch pipeline should be part of the template. It's specific to ssh. `pipelines/fetch.ttl` could be also removed.

File: package.json [hunk 2]
```diff
@@ -1,27 +1,34 @@
...
-    "readable-stream": "^3.6.0"
+    "barnard59": "^1.1.1",
+    "barnard59-csvw": "^1.0.0",
+    "barnard59-formats": "^1.2.0",
+    "barnard59-ftp": "^1.0.1",
+    "barnard59-graph-store": "^1.0.1",
+    "barnard59-rdf": "^1.1.1",
+    "jsonld": "^4.0.1",
+    "rdf-ext": "^2.0.1",
+    "rdf-stream-to-dataset-stream": "^1.0.0"
```
Comment (bergos) [inline package.json]: - `jsonld` is no longer required (was in there as workaround for the buggy version 5.x)
- `rdf-stream-to-dataset-stream` is not required
- `barnard59-ftp` could be removed if the fetch pipeline gets removed

**Review by bergos — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ktk) [reply package.json]: I was debating about it. We might say that fetch is only part of our template in GitLab but not for this public one on GitHub

File: package.json (hunk 2 above)
Comment (ktk) [reply package.json]: Cleaned up in https://github.com/zazuko/xrm-csvw-workflow/pull/16/commits/8da00287991833386d50ba07913e084a2d95ac14

File: package.json (hunk 1 above)
Comment (bergos) [reply package.json]: Not sure about GitLab, but I'm for removing it here. Less files -> less confusing -> higher acceptance.

Comment (ktk) [reply package.json]: It's the internal one @ gov, will remove it here.

Comment (ktk) [reply package.json]: Removed in https://github.com/zazuko/xrm-csvw-workflow/pull/16/commits/aecaff3d46dacca7da8a3520038b344651c97e89

**Review by bergos — APPROVED** [review]

**Pull request merged**